source/standalone/workflows/rsl_rl/play.py

```diff
@@ -59,22 +66,33 @@ def main():
     )
     agent_cfg: RslRlOnPolicyRunnerCfg = cli_args.parse_rsl_rl_cfg(args_cli.task, args_cli)
 
-    # create isaac environment
-    env = gym.make(args_cli.task, cfg=env_cfg)
-    # wrap around environment for rsl-rl
-    env = RslRlVecEnvWrapper(env)
-
     # specify directory for logging experiments
     log_root_path = os.path.join("logs", "rsl_rl", agent_cfg.experiment_name)
     log_root_path = os.path.abspath(log_root_path)
     print(f"[INFO] Loading experiment from directory: {log_root_path}")
     resume_path = get_checkpoint_path(log_root_path, agent_cfg.load_run, agent_cfg.load_checkpoint)
-    print(f"[INFO]: Loading model checkpoint from: {resume_path}")
+    log_dir = os.path.dirname(resume_path)
 
+    # create isaac environment
+    env = gym.make(args_cli.task, cfg=env_cfg, render_mode="rgb_array" if args_cli.video else None)
+    # wrap for video recording
+    if args_cli.video:
+        video_kwargs = {
+            "video_folder": os.path.join(log_dir, "videos"),
```

Comment:
Maybe we should have separate video folders for training and playing? Otherwise their names will overwrite each other.

Reply:
How about using `videos/<TIMESTAMP>`? This will also allow recording more than one video by running the play script multiple times.

Reply:
Sounds fair 👀 ... But for training the directory is already with the timestamp, right? Might be redundant/confusing to have the timestamp there twice when you train (especially since the two time-stamps won't match)?

Reply:
Yes, it would look something like this: `logs/skrl/reach_franka/2024-07-30_16-29-28/videos/2024-07-31_10-25-02/rl-video-step-0.mp4`. It might be a bit confusing, but I haven't come up with a better naming rule yet...

Reply:
or we simply split into `videos/train` and `videos/play` ?

Reply:
Sounds good to me. @j3soon can you change the train/play scripts accordingly? :)